### PR TITLE
Breaking: Remove ARMv7 building of the Docker image

### PR DESCRIPTION
**Pull request by stumpylog**
## Proposed change

Removes the workflow option to build the ARMv7 Docker image and removes the installation of our pre-built packages, instead using those already built by their maintainers or building them on the fly (hence the addition of libpq-dev).

This is a breaking change for users on the ARMv7 platform, they will no longer be able to directly pull the image.  They may yet be able to build it themselves, at least until the dependencies are updated, etc.

- [x] Check if the docs mention the supported platforms and update those.

~Draft for now, until we're satisfied with v1.17.x.  Dependency unlocking will happen in a different PR.~

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (please explain):

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [x] If applicable, I have included testing coverage for new code in this PR, for [backend](https://docs.paperless-ngx.com/development/#testing) and / or [front-end](https://docs.paperless-ngx.com/development/#testing-and-code-style) changes.
- [x] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [x] If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).
- [x] I have run all `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).
- [x] I have made corresponding changes to the documentation as needed.
- [x] I have checked my modifications for any breaking changes.

### CODE REVIEW

**Comment by stumpylog** [comment]
That's generally correct.  There's more information in the announcement: https://github.com/paperless-ngx/paperless-ngx/discussions/3940

File: docs/faq.md [hunk 1]
```diff
@@ -92,6 +92,13 @@ the python requirements do not have precompiled packages for ARM /
 ARM64. Installation of these will require additional development
 libraries and compilation will take a long time.
 
+!!! note
+
+    For ARMv7 systems, paperless may still function, but it could require
```
Comment (shamoon) [inline docs/faq.md:97]: Is it worth clarifying what this is e.g. 32bit ARM?

**Review by shamoon — APPROVED** [review]
Thanks for all the work, here's hoping this streamlines things a bit

File: docs/faq.md (hunk 1 above)
Comment (stumpylog) [reply docs/faq.md]: Tweaked to add that

**Pull request merged**